simul/platform/mininet/setup_servers.sh

```diff
@@ -21,59 +21,64 @@ fi
 
 rm -f $KEYS
 for s in $SERVERS; do
-	echo Starting to install on $s
+	port=22
```

Comment:
Same discussion as above: can you have `-p ${port}` as a string, so if no port is given the default port is figured out automatically, possibly from `~/.ssh/config`?